spec/helpers/session.rb

```diff
@@ -0,0 +1,23 @@
+module SessionHelpers
+
+  def sign_up
+    user = build :user
+    visit '/'
+    click_link 'Sign up'
+    fill_in :email, with: user.email
+    fill_in :username, with: user.username
+    fill_in :password, with: user.password
+    click_on 'Sign up'
+  end
+
+  def sign_in
+    sign_up
```

Comment:
You don't need to sign up here - just create the user in the database (e.g. using FactoryGirl)